conda/gateways/connection/session.py

```diff
@@ -62,10 +74,32 @@ def __call__(cls):
             return session
 
 
-class CondaSession(Session, metaclass=CondaSessionType):
+def only_cache_json_jlap(response):
+    return response.url.endswith(('repodata.json', 'repodata.jlap'))
```

Comment:
Since this is only used in one unique place, can we inline it as a lambda function?